### DISCUSSION TITLE
Pemilu 2024

### DISCUSSION THREAD

**Discussion by masrokhim** [community/community#119766] [Discussions]
### Select Topic Area

Question

### Body

Mas Adi, perkenalkan saya Rokhim - Malang, ijin, saya telah mencoba repository jenengan Pemilu 2024, namun saya tidak berhasil login, apakah ada yang perlu diseting, atau tahapan untuk install aplikasinya. terima kasih.

**Comment by Akash1134** [comment]
Hi @masrokhim , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
